src/pytest_servers/gcs.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import time
+
+import pytest
+import requests
+
+from .utils import get_free_port
+
+logger = logging.getLogger(__name__)
+
+GCS_DEFAULT_PORT = 4443
+
+
+@pytest.fixture(scope="session")
+def fake_gcs_server(docker_client):
+    if docker_client is None:
+        logger.warning(
+            "fake-gcs-server cannot run because docker is not available."
+        )
+        yield None
+        return
+
+    # To enable support for signed URLs, we need to provide
+    # -public-host and -external-url, so we need to get a free
+    # port beforehand
+    port = get_free_port()
+    command = f"-scheme http -public-host http://localhost:{port} -external-url http://localhost:{port}"  # noqa: E501
```

Comment:
There is really no reason to make an exception for this long line. Just split it in two.